Aleksand Zorin/index.js

```diff
@@ -0,0 +1,128 @@
+function Food(type, prise, calorie) {
+    this.type = type;
+    this.prise = prise;
+    this.calorie = calorie;
+}
+
+Food.prototype.getPrise = function() {
+    return this.prise;
+}
+
+Food.prototype.getCalorie = function() {
+    return this.calorie;
+}
+
+Food.prototype.toString = function() {
+    return this.type;
+}
+
+function Hamburger(size,stuffing) {
+    this.size = size.type;
+    this.stuffing = stuffing.type;
+    Food.call(this,"Hamburger", size.prise + stuffing.prise, size.calorie + stuffing.calorie);
+}
+
+Hamburger.prototype = Food.prototype;
+
+Hamburger.prototype.getSize = function() {
+    return this.size;
+}
+
+Hamburger.prototype.getStuffing = function() {
+    return this.stuffing;
+}
+
+Hamburger.prototype.toString = function() {
+    return this.type + " (" + this.size + ", " + this.stuffing + ")";
+}
+
+function Order() {
+    this.foods = [];
+}
+
+Order.prototype.add = function(food) {
+    if (Object.isFrozen(this.foods)) {
+        console.log("you cannot edit an already paid order");
+        return;
+    }
+    this.foods.push(food);
+    
+    return this;
```

Comment:
@Zorin-Alex Since your app don't have UI part it's better to show message after successful added/removed item